crates/bevy_sprite/src/lib.rs

```diff
@@ -49,21 +54,36 @@ impl Plugin for SpritePlugin {
             .add_asset::<TextureAtlas>()
             .register_type::<Sprite>()
             .add_system_to_stage(CoreStage::PostUpdate, sprite_system.system())
+            .add_system_to_stage(
+                CoreStage::PostUpdate,
+                frustum_culling::frustum_culling_sprites.system(),
```

Comment:
nit: since the module is called `frustum_culling`, it feels somewhat redundant to have the same name appear in the function. Perhaps `frustum::culling::{sprites, atlases}`?